examples/sdk-examples/src/examples/dashboardComponent/DashboardComponentWithUseDispatchDashboardCommandSrc.tsx

```diff
@@ -0,0 +1,167 @@
+    /**
+     * Select the attribute filter's local identifier from the filter's display form.
+     */
+    const restaurantCategoryFilterLocalId = useDashboardSelector(
+        selectFilterContextAttributeFilterByDisplayForm(
+            uriRef("/gdc/md/xms7ga4tf3g3nzucd8380o2bev8oeknp/obj/2197"),
+        ),
+    )?.attributeFilter.localIdentifier;
+
+    /**
+     * Callbacks for the buttons.
+     *
+     * On {@link changeRestaurantCategoryFilterSelection}, {@link resetRestaurantCategoryFilterSelection} and {@link changeDashboardDateFilterSelection}
+     * examples you can see how to call the command created by the useDispatchDashboardCommand hook with payload properties.
+     *
+     * {@link resetDashboardDateFilter} is an example on how to dispatch command without these properties.
+     */
+    const changeRestaurantCategoryFilterSelection = () => {
+        restaurantCategoryFilterLocalId &&
+            changeAttributeFilterSelectionCmd(
+                restaurantCategoryFilterLocalId,
+                {
+                    uris: ["/gdc/md/xms7ga4tf3g3nzucd8380o2bev8oeknp/obj/2196/elements?id=1"],
+                },
+                "IN",
+            );
+    };
+
+    const resetRestaurantCategoryFilterSelection = () => {
+        restaurantCategoryFilterLocalId && resetAttributeFilter(restaurantCategoryFilterLocalId);
+    };
+
+    const changeDashboardDateFilterSelection = () => {
+        changeDateFilterSelectionCmd("relative", "GDC.time.year", -3, 0);
+    };
+
+    const resetDashboardDateFilter = () => {
+        resetDateFilter();
+    };
+
+    return (
+        <div>
+            <button onClick={changeRestaurantCategoryFilterSelection}>
+                Change Restaurant category selection
+            </button>
+            <button onClick={resetRestaurantCategoryFilterSelection}>Reset Restaurant category filter</button>
+            <button onClick={changeDashboardDateFilterSelection}>Change date filter selection</button>
+            <button onClick={resetDashboardDateFilter}>Clear date filter selection</button>
+        </div>
+    );
+};
+
+class LocalPlugin extends DashboardPluginV1 {
+    public readonly author = "John Doe";
+    public readonly displayName = "Sample local plugin";
+    public readonly version = "1.0";
+
+    public register(
+        _ctx: DashboardContext,
+        customize: IDashboardCustomizer,
+        _handlers: IDashboardEventHandling,
+    ): void {
+        customize.customWidgets().addCustomWidget("myCustomWidget", MyCustomWidget);
+        customize.layout().customizeFluidLayout((_layout, customizer) => {
+            customizer.addSection(
+                0,
+                newDashboardSection(
+                    "Buttons to dispatch filter commands",
+                    newDashboardItem(
+                        newCustomWidget("myWidget1", "myCustomWidget", {
+                            // specify which date data set to used when applying the date filter to this widget
```

Comment:
This object is not necessary here since the custom widget does not care about the filters (it has no execution in it).